style/styles_social.css

```diff
@@ -0,0 +1,222 @@
+.social {
+  display: flex;
+  flex-direction: column;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.social__title {
+  align-self: flex-end;
+  margin: 0;
+  font-size: 18px;
+  font-family: "Gotham Pro Bold", sans-serif;
+  opacity: 0.3;
+  padding-bottom: 8px;
+}
+
+.social__block {
+  display: flex;
+}
+
+.social-eight {
+  display: none;
+}
+
+.social__pic {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+ @media (max-width: 1100px) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)